mmengine/optim/optimizer/optimizer_wrapper.py

```diff
@@ -161,6 +162,14 @@ def __init__(self,
         # the loss factor will always be the same as `_accumulative_counts`.
         self._remainder_counts = -1
 
+        # add a tensor to the optimizer to retrieve global lr
+        if hasattr(self.optimizer, 'defaults'):
```

Comment:
Will the optimizer not have the `default` attribute?

Reply:
in some test cases, a mock class may not have defaults

Reply:
We should not make the code comply with unit tests. If the unit tests are unreasonable, they can be modifed.

Reply:
solved, modified test code accordingly